app/views/reset_password_mailer/reset_password.text.erb

```diff
@@ -1,6 +1,8 @@
 Hi <%= @email_address.user.casual_name %>,
 
-We received a request to reset your password.  To reset your password, please follow the link below:
+Someone just reported that they can’t log in to your OpenStax account.  If this wasn’t you, please ignore this message.
+
+To reset your password, please follow the link below:
 
 <%= reset_password_url code: @reset_password_code %>
 
```

Comment:
When you add the social links to this email, can you change the mailer to be "sign_in_help_mailer" or similar? (since it will no longer just be "reset_password")